pkg/api/authentication.go

```diff
@@ -1,159 +0,0 @@
-package api
```

Comment:
This is all dead code since we don't allow writes on V2, which was what this was used for.